actortoken/client.go

```diff
@@ -9,8 +9,25 @@ import (
 	"github.com/clerk/clerk-sdk-go/v2"
 )
 
+//go:generate go run ../cmd/gen/main.go
```

Comment:
This would be better written as
```
//go:generate gen
```
but I couldn't figure out how to build the package and add it to the PATH. I should probably give it more time at some point.